### PR TITLE
Updated UG for Meeting Section

### PR DESCRIPTION
**Pull request by Christopher-LM**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -751,13 +751,13 @@ Listed all meetings
...
-Edits an existing meeting detail that is in the list.
+Edits an existing meeting in the meeting list. The existing meeting to be editted will have its parameters updated with the new values provided by the user.
 
 - Command: `edit-m`
-- Format: `edit-m <INDEX_NUMBER> b/BIDDER_ID p/PROPERTY_ID v/VENUE t/DATE`
+- Format: `edit-m <INDEX_NUMBER> b/BIDDER_ID p/PROPERTY_ID v/VENUE d/DATE s/STARTTIME e/ENDTIME`
 
 > - Edits the meeting at the specified `INDEX_NUMBER`, which refers to the index shown on the displayed meeting list. The index must be a positive integer 1, 2, 3...
-
+More than one parameter can be updated for the existing meeting to be editted.
```
Comment (Marcon2509) [inline docs/UserGuide.md]: this refer to dianne property got a line that says smth like at least one optional parameter must be given

File: docs/UserGuide.md [hunk 2]
```diff
@@ -776,14 +776,39 @@ Venue: eunos
...
+Successfully sorted meeting
+```
 
 ## Find an Existing Meeting
 
 Finds an existing meeting detail that is in the list.
 
 - Command: `find-m`
-- Format: `find-m <INDEX> b/BIDDER_ID p/PROPERTY_ID v/VENUE t/DATE s/STARTTIME v/ENDTIME`
+- Format: `find-m <INDEX> b/BIDDER_ID p/PROPERTY_ID v/VENUE d/DATE s/STARTTIME e/ENDTIME`
```
Comment (Marcon2509) [inline docs/UserGuide.md]: INDEX_NUMBER not INDEX

File: docs/UserGuide.md [hunk 3]
```diff
@@ -899,11 +924,12 @@ Picture Example:
...
 | :---| :--- |
-| add-m q/MEETING_TYPE b/BIDDER_ID p/PROPERTY_ID v/VENUE t/DATE s/STARTTIME v/ENDTIME | add-m q/v b/B1 p/P1 v/2 ALBERT PARK d/11-12-2021 s/12:30 e/13:00  |
+| add-m q/MEETING_TYPE b/BIDDER_ID p/PROPERTY_ID v/VENUE d/DATE s/STARTTIME e/ENDTIME | add-m q/v b/B1 p/P1 v/2 ALBERT PARK d/11-12-2021 s/12:30 e/13:00  |
 | delete-m <INDEX> | delete-m 1 |
-| find-m <INDEX> b/BIDDER_ID p/PROPERTY_ID v/VENUE t/DATE s/STARTTIME v/ENDTIME | find-m b/B1 p/P1|
+| find-m <INDEX> b/BIDDER_ID p/PROPERTY_ID v/VENUE d/DATE s/STARTTIME e/ENDTIME | find-m b/B1 p/P1|
 | list-m | list-m |
-
+| edit-m <INDEX> b/BIDDER_ID p/PROPERTY_ID v/VENUE d/DATE s/STARTTIME e/ENDTIME | edit-m  b/B1 p/P1 |
+| sort-m o/<ORDER> | sort-m o/asc |
```
Comment (Marcon2509) [inline docs/UserGuide.md]: INDEX_NUMBER not INDEX and add ther `e/ENDTIME` like the other tables

**Review by Marcon2509 — APPROVED** [review]
Can edit the things first

File: docs/UserGuide.md (hunk 1 above)
Comment (Christopher-LM) [reply docs/UserGuide.md]: Fixed

File: docs/UserGuide.md (hunk 2 above)
Comment (Christopher-LM) [reply docs/UserGuide.md]: Fixed for all INDEX i could find

**Review by dianneloh9 — APPROVED** [review]

**Pull request merged**